src/migrations/20200303162506-create-group-model.js

```diff
@@ -0,0 +1,45 @@
+'use strict'
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable('group', {
+      id: {
+        type: Sequelize.UUID,
+        primaryKey: true,
+        defaultValue: Sequelize.UUIDV4
+      },
+      productId: {
+        type: Sequelize.UUID,
+        allowNull: false,
+        references: {
+          model: 'product',
+          key: 'id'
+        }
+      },
+      type: {
+        type: Sequelize.ENUM,
+        values: [
+          'PAIR',
+          'GROUP'
+        ],
+        allowNull: false
+      },
+      deadline: {
```

Comment:
Let's rename this to "expiresAt".

Reply:
By mistake I created the expiresAt but forgot to delete this, already fixed.